dask/dataframe/io/tests/test_csv.py

```diff
@@ -720,13 +721,15 @@ def test_read_csv_sensitive_to_enforce():
 def test_read_csv_compression(fmt, blocksize):
     if fmt not in compress:
         pytest.skip("compress function not provided for %s" % fmt)
+    suffix = {"gzip": ".gz", "bz2": ".bz2", "zip": ".zip", "xz": ".xz"}.get(fmt, "")
```

Comment:
We should somehow test for no compression.

A comment should be added to say that this test if effectively using "infer" and the extension, or we could test with both infer and explicit (which you already did above, so not necessary).

Reply:
Okay - I changed the `parametrize` layout a bit and added a non-compression test.  I also added a brief comment to explain that we are relying on `compression="infer"`